### PR TITLE
[CR] Add linothorax and laminated cotton

### PR DESCRIPTION
**Pull request by avianArsonist**
#### Summary
Content "Linothorax armor/material additions"

#### Purpose of change

Adds more early-to-mid-game armor.
#### Describe the solution
Adds the linothorax, a corselet made from laminated linen, or possibly twined linen, or less probably leather. I'm going with laminated linen, represented in-game by laminated cotton.  It can be crafted, found in museums, or sold by Cody,

Adds two items, linothorax and drying linothorax, the latter of which transforms into the former after two days to represent glue curing. Adds a crafting recipe for the drying linothorax, found in Studies in Historic Armorsmithing. Adds laminated cotton as a material. Adds the linothorax to Cody's armor selling spawngroup, and to museum armor itemgroups containing the lorica segmentata.
#### Describe alternatives you've considered

Use fake thickness of cotton to simulate lamination, include glue curing in the recipe crafting time.
#### Testing

Spawned into a new world, spawned linothorax, refitted and wore it. Spawned drying linothorax, waited two days, it changed into linothorax successfully. Spawned materials, crafted drying linothorax from materials, waited for it to transform, transformed successfully. Checked that recipe was missing without skill book; spawned skill book; checked recipe was present.
#### Additional context

Screenshots:
image
image

Thoughts:
I'm not at all certain on laminated cotton's stats, most of the ones I don't fully understand are copied from cotton and the exact protective values aren't quite pinned down. Additionally, the linothorax should be made from laminated linen, but as linen doesn't exist in-game currently it's made from laminated cotton for now, and uses "cloth" in the description in place of "linen".

Sources:
 https://gregorysaldrete.com/hompage/linothorax-project/
https://www.youtube.com/watch?v=SLBMupbqo2I
https://www.youtube.com/watch?v=r4SC3DLclqY
https://www.youtube.com/watch?v=4qyGge-laQY

### CODE REVIEW

File: data/json/items/armor/suits_protection.json [hunk 1]
```diff
@@ -2330,6 +2330,90 @@
       "menu_text": "Adjust tighter"
     }
   },
+  {
+    "id": "linothorax",
+    "type": "ARMOR",
+	"category": "armor",
+    "name": { "str": "linothorax", "str_pl": "linothorakes" },
+    "description": "Ancient Greek armor made from laminated cloth. Lighter and more comfortable than bronze while providing similar protection.",
+    "weight": "5000 g",
```
Comment (Meandering-Breviger) [inline data/json/items/armor/suits_protection.json:2339]: I'd change this to "5 kg" as it's a nice even number but all in all awesome pr.
Also for your pr validation test error change `Category "content"` to something like `Content "Linothorax armor additions"`

File: data/json/materials.json [hunk 2]
```diff
@@ -553,6 +553,37 @@
...
+    "fire_resist": 1,
+    "elec_resist": 2,
+    "chip_resist": 9,
+    "breathability": "AVERAGE",
+	"wind_resist": 80,
+    "repaired_with": "cotton_patchwork",
+    "salvaged_into": "sheet_cotton_patchwork",
+    "dmg_adj": [ "ripped", "torn", "shredded", "tattered" ],
+    "bash_dmg_verb": "ripped",
+    "cut_dmg_verb": "cut"
```
Comment (Meandering-Breviger) [inline data/json/materials.json:580]: This needs a , to fix the error.

File: data/json/items/armor/suits_protection.json [hunk 3]
```diff
@@ -2330,6 +2330,90 @@
...
+	"to_hit": -5,
+    "bashing": 2,
+    "material": [ "cotton_laminated" ],
+	"symbol": "[",
+    "looks_like": "gambeson",
+    "color": "light_gray",
+	"warmth": 15,
+	"longest_side": "76 cm",
+    "material_thickness": 10,
+    "flags": [ "VARSIZE", "STURDY", "OUTER" ]
```
Comment (Meandering-Breviger) [inline data/json/items/armor/suits_protection.json:2352]: This also needs a ,

File: data/json/recipes/armor/suit.json [hunk 4]
```diff
@@ -1700,6 +1700,26 @@
...
+    "subcategory": "CSC_ARMOR_SUIT",
+    "skill_used": "tailor",
+    "difficulty": 3,
+	"skills_required": [ [ "fabrication", 3 ], [ "chemistry", 1 ] ],
+    "time": "4 h",
+    "autolearn": true,
+	"book_learn": [ [ "textbook_armwest", 2 ], [ "tailor_portfolio", 2 ] ],
+    "using": [ ["strap_small", 2], [ "fastener_large", 2 ], [ "fastener_large", 1 ], [ "sewing_standard", 10 ] ],
+    "qualities": [ { "id": "LEATHER_AWL", "level": 1 }, { "id": "FABRIC_CUT", "level": 1 } ],
+    "proficiencies": [ { "proficiency": "prof_closures" } ]
```
Comment (Meandering-Breviger) [inline data/json/recipes/armor/suit.json:1717]: , here also

File: data/json/recipes/armor/suit.json [hunk 5]
```diff
@@ -1700,6 +1700,23 @@
...
+    "result": "drying_linothorax",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_SUIT",
+    "skill_used": "tailor",
+    "difficulty": 3,
+    "skills_required": [ [ "fabrication", 3 ], [ "chemistry", 1 ] ],
+    "time": "4 h",
+    "autolearn": true,
```
Comment (TheShadowFerret) [inline data/json/recipes/armor/suit.json:1713]:
```suggestion
```
> In little more than two decades, Alexander the Great of Macedon (356-323 B.C.) conquered by military force nearly the entirety of the known world. Despite the fact that he led one of the most successful armies of all time, **surprisingly little is understood** about the main type of body armor that both Alexander and many of his men apparently wore.

This is not something you'd be able to just figure out.

Comment (avianArsonist) [reply data/json/recipes/armor/suit.json:1713]: I feel like the general concept of laminated linen armor might be possible to intuit (not 100% sure on that), but either way the specific garment probably wouldn't be; I'll change it to book-only

Comment (avianArsonist) [reply data/json/recipes/armor/suit.json]: Now that I think about it, it should probably be removed from the clothing designer's portfolio

**Review by bombasticSlacks — APPROVED** [review]
Haven't tested in game but the changes look good to me

File: data/json/materials.json [hunk 6]
```diff
@@ -553,6 +553,37 @@
       { "fuel": 1, "smoke": 1, "burn": 2 }
     ]
   },
+  {
+    "type": "material",
+    "id": "cotton_laminated",
+    "name": "Laminated Cotton",
+	"//": "This should really be linen",
```
Comment (dseguin) [inline data/json/materials.json:560]:
```suggestion
    "//": "This should really be linen",
```

File: data/json/materials.json [hunk 7]
```diff
@@ -553,6 +553,37 @@
...
+    "density": 10,
+    "specific_heat_liquid": 0.02,
+    "specific_heat_solid": 0.02,
+    "latent_heat": 205,
+    "soft": true,
+	"reinforces": true,
```
Comment (dseguin) [inline data/json/materials.json:566]:
```suggestion
    "reinforces": true,
```

File: data/json/materials.json [hunk 8]
```diff
@@ -553,6 +553,37 @@
...
+    "bash_resist": 1.8,
+    "cut_resist": 1.8,
+    "bullet_resist": 1.6,
+    "acid_resist": 3,
+    "fire_resist": 1,
+    "elec_resist": 2,
+    "chip_resist": 9,
+    "breathability": "AVERAGE",
+	"wind_resist": 80,
```
Comment (dseguin) [inline data/json/materials.json:575]:
```suggestion
    "wind_resist": 80,
```

File: data/json/materials.json [hunk 9]
```diff
@@ -553,6 +553,37 @@
...
+    "repaired_with": "cotton_patchwork",
+    "salvaged_into": "sheet_cotton_patchwork",
+    "dmg_adj": [ "ripped", "torn", "shredded", "tattered" ],
+    "bash_dmg_verb": "ripped",
+    "cut_dmg_verb": "cut",
+	"burn_data": [
```
Comment (dseguin) [inline data/json/materials.json:581]:
```suggestion
    "burn_data": [
```

File: data/json/materials.json [hunk 10]
```diff
@@ -553,6 +553,37 @@
...
+      { "fuel": 2, "smoke": 1, "burn": 1, "volume_per_turn": "1100 ml" },
+      { "fuel": 2, "smoke": 1, "burn": 1 },
```
Comment (dseguin) [inline data/json/materials.json:583]:
```suggestion
      { "fuel": 2, "smoke": 1, "burn": 1 },
```

**Review by dseguin — COMMENTED** [review]
Just some minor linting for the json style check

**Comment by Drew4484** [comment]
Is this still being worked on?

**Comment by Kantonine** [comment]
Anyone still interested in this? a nearly 2 year old pr

**Comment by Kantonine** [comment]
Welp, sad to see this collect dust

**Comment by anothersimulacrum** [comment]
I'm going to close this as stale. It hasn't seen activity in over a year. @Kantonine If you would like to adopt this PR, it's possible.

**Comment by Zireael07** [comment]
Copy the code and open a  new PR

**Comment by anothersimulacrum** [comment]
Please add `co-authored-by: avianArsonist <91783002+avianArsonist@users.noreply.github.com>` to the commit description when you do so.

**Comment by Kantonine** [comment]
> Please add `co-authored-by: avianArsonist <91783002+avianArsonist@users.noreply.github.com>` to the commit description when you do so. (For attribution reasons, this is not really optional)

Of course! Would leave a bad taste in my mouth to do so otherwise

**Pull request closed without merging**